scene/gui/graph_edit.h

```diff
@@ -98,6 +98,7 @@ class GraphEdit : public Control {
 
 	bool dragging;
 	bool just_selected;
+	bool move_signal_sent;
```

Comment:
I think this should be renamed to `moving_selection` similar to the boolean from animation_bezier_editor/track editor.

Reply:
Updated